samples/msgext-ai-sentiment-analysis/nodejs/env/.env.local

```diff
@@ -0,0 +1,22 @@
+# This file includes environment variables that can be committed to git. It's gitignored by default because it represents your local development environment.
+
+# Built-in environment variables
+TEAMSFX_ENV=local
+
+# Generated during provision, you can also add your own variables. If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
+BOT_ENDPOINT=
+BOT_DOMAIN=
+AAD_APP_CLIENT_ID=
+AAD_APP_OBJECT_ID=
+AAD_APP_TENANT_ID=
+AAD_APP_OAUTH_AUTHORITY=
+AAD_APP_OAUTH_AUTHORITY_HOST=
+TEAMS_APP_ID=
+TEAMS_APP_TENANT_ID=
+AAD_APP_ACCESS_AS_USER_PERMISSION_ID=
+MICROSOFT_APP_TYPE=
+MICROSOFT_APP_TENANT_ID=
+API_KEY=
+RESOURCE_SUFFIX=
+AZURE_SUBSCRIPTION_ID=
+AZURE_RESOURCE_GROUP_NAME=
```

Comment:
Do we need all of these values?
How these will generate ?

No steps are mentioned in readme to update these values.

Reply:
fixed